.travis.yml

```diff
@@ -14,6 +14,10 @@ env:
 language: python
 python:
   - "2.7"
+matrix:
+  include:
+    - python: "3.4"
+      env: TEST_SUITE=mypyer
 # command to run tests
 script:
   - ./tools/travis/$TEST_SUITE
```

Comment:
Oops, yeah I should clarify a bit more what I had in mind:

* We should move most of the script to something like `tools/run-mypy` or `check-mypy` so that it's easy to run in a development environment.  The `tools/travis/` path is intended to just be the little wrappers or even symlinks that we need to run tests in Travis CI; the files are named based on what we want to appear in the Travis CI output as the name of the test, which I think in this case probably should be "mypy".  I suppose what's confusing here is that `tools/travis/py3k` doesn't fit the pattern; we should probably fix that by moving the script to a better name like `tools/check-py3` (and making the `tools/travis` path a symlink or thin wrapper).
* `tools/travis/mypy` would just be a thin wrapper around that, and I don't expect to be referred to often.